crates/roc_std/src/roc_list.rs

```diff
@@ -676,7 +679,10 @@ where
         let ptr = self.ptr_to_refcount();
         unsafe {
             let value = std::ptr::read(ptr);
-            std::ptr::write(ptr, Ord::max(0, ((value as isize) + 1) as usize));
+            // Only safe to write to the pointer if it is not constant (0)
+            if value != 0 {
```

Comment:
Don't know our coding style, but wouldn't it be better to put this magic constant 0 into a named constant?

Reply:
yes

Reply:
updated